### PR TITLE
Add a changeset to Delete coded obs with null values from the summary…

### PR DESCRIPTION
**Pull request by ssmusoke**
… page - most of these seem to be health education related concepts which are not on the summary page. This issue was faced in Kisugu during the field testing of MCH forms

@dkayiwa I think this looks better :-)

### CODE REVIEW

**Comment by dkayiwa** [comment]
@ssmusoke it does not just look better, it is rather the best!!! 😊 👍

File: api/src/main/resources/liquibase.xml [hunk 1]
```diff
@@ -8,6 +8,12 @@
         See http://www.liquibase.org/manual/home#available_database_refactorings
         for a list of supported elements and attributes
     -->
+	<changeSet id="ugandaemr-07032017-1958" author="ssmusoke">
+		<comment>Delete coded obs with null values from the summary page - most of these seem to be health education related concepts which are not on the summary page</comment>
+		<sql>
+			DELETE FROM obs WHERE concept_id IN (99169, 99170, 99171,99172, 99173,99174, 99175,99176,99177,99178,99179,99180,99181) AND value_coded IS NULL
```
Comment (dkayiwa) [inline api/src/main/resources/liquibase.xml]: Do you have any idea how these got saved in the first place, with just NULL?
Or was the value saved or accidentally saved in another field like value_numeric, value_boolean, etc?

Comment (ssmusoke) [inline api/src/main/resources/liquibase.xml]: @dkayiwa I have no idea how these got entered, however this database was migrated so I suspect that may have happened in the 1.6.x versions of the forms. Its the first time that I am seeing this error and took about half a day to chase down. 

We are finding that the validation checks in the 2.x migration are more stringent than during 1.x - so I expect to see a lot of these happening

Comment (dkayiwa) [inline api/src/main/resources/liquibase.xml]: Oh i see!
Fair enough.

**Pull request merged**